code/modules/battlemonsters/items/wrapped.dm

```diff
@@ -0,0 +1,47 @@
+/obj/item/battle_monsters/wrapped
+	name = "battle monsters starterpack"
+	desc = "A battle monsters 52 card deck starterpack. Contains 52 basic cards and a lifetime of sadness."
+	icon_state = "pack1"
+	var/contained_cards = 52
+	var/rare_limiter = 1000
+
+/obj/item/battle_monsters/wrapped/attack_self(mob/user)
+	var/list/deck_data = list()
+	for(var/i=1,i < contained_cards,i++)
+		CHECK_TICK //This stuff is a little intensive I think.
```

Comment:
👀

Reply:
No, it's just that I feel like something like this shouldn't be expensive enough that a C_T is required. Does the generation code do a lot of string processing or something?

Reply:
I'm not entirely sure. It just gets a random element from a list using pickweight 3 times, 52 times.